release_notes/addtl-release-notes.adoc

```diff
@@ -0,0 +1,90 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="addtl-release-notes"]
+= Additional release notes
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-microshift.adoc[]
+include::_attributes/servicebinding-document-attributes.adoc[]
+:context: addtl-release-notes
+
+Release notes for additional related components and products not included in the core xref:../release_notes/ocp-release-notes.adoc#ocp-release-notes[{product-title} {product-version} release notes] are available in the following documentation.
+
+[IMPORTANT]
+====
+The following release notes are for downstream Red Hat products only; upstream or community release notes for related products are not included.
+====
+
+[cols="4,7a",options="header"]
+|===
+|Component or product |Release note documentation
+
+|AWS Load Balancer (ALB) Operator
+|xref:../networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc#aws-load-balancer-operator-release-notes[AWS Load Balancer Operator release notes]
+
+|Compliance Operator
+|xref:../security/compliance_operator/compliance-operator-release-notes.adoc#compliance-operator-release-notes[Compliance Operator release notes]
+
+|{rh-dev-hub} Operator
+|link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.3/html/release_notes/index[Release notes]
+
+|External DNS Operator
+|xref:../networking/external_dns_operator/external-dns-operator-release-notes.adoc#external-dns-operator-release-notes[External DNS Operator release notes]
+
+|File Integrity Operator
+|xref:../security/file_integrity_operator/file-integrity-operator-release-notes.adoc#file-integrity-operator-release-notes-v0[File Integrity Operator release notes]
+
+|{descheduler-operator}
+|xref:../nodes/scheduling/descheduler/nodes-descheduler-release-notes.adoc#nodes-descheduler-release-notes[{descheduler-operator} release notes]
+
+|{logging-uc}
+|xref:../observability/logging/logging_release_notes/logging-5-9-release-notes.adoc#logging-5-9-release-notes[{logging-uc} -> Release notes]
```

Comment:
```suggestion
|xref:../observability/logging/logging_release_notes/logging-5-9-release-notes.adoc#logging-5-9-release-notes[{logging-uc} -> Release notes]
```

Reply:
Why does the Logging release notes have an arrow? 🤔 I'm guessing there's a reason for this but jw.

Reply:
+1. Even though it doesn't match the page title, I think you could still just use "Logging release notes" here. Or there might be more specific operators to list. Maybe "OpenShift Logging Operator"? Though it looks like there's also a Loki Operator, so not certain. Probably need a logging writer to weigh in.

Reply:
The arrow was sorta vestigial from an earlier iteration / different thought process on the layout. Dropped it now!

Reply:
If we do want to get more specific about any related Operator names, lmk.